config/samples/translator-binding.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: ibmcloud.ibm.com/v1alpha1
 kind: Binding
 metadata:
   name: binding-translator
+  namespace: name2
```

Comment:
Can we get rid of namespace here so it can be used in any namespace ?

Reply:
I removed the namespaces from the 2 samples